ingestion_tools/scripts/data_validation/fixtures/path.py

```diff
@@ -69,16 +70,20 @@ def run_meta_file(run_dir: str, filesystem: FileSystemApi) -> str:
 
 
 @pytest.fixture(scope="session")
-def frames_dir(run_dir: str, tiltseries_metadata: Dict[str, Any], filesystem: FileSystemApi) -> str:
+def frames_dir(run_dir: str, filesystem: FileSystemApi) -> str:
     """[Dataset]/[ExperimentRun]/Frames"""
     dst = f"{run_dir}/Frames"
     if filesystem.s3fs.exists(dst):
```

Comment:
looks like we are using the `filesystem.s3fs.exists` in multiple places. So, we should support that as a method in `filesystem`.